src/openrct2/game.h

```diff
@@ -150,7 +150,7 @@ extern float gDayNightCycle;
 extern bool gInUpdateCode;
 extern sint32 gGameCommandNestLevel;
 extern bool gGameCommandIsNetworked;
-extern char gCurrentlyLoadedPath[260];
+extern char gCurrentlyLoadedPath[_MAX_PATH];
```

Comment:
I would need to add an include to the header file, would that be alright? What include is the one that holds "MAX_PATH"? (I'm asking which one cause I know that i've seen some source files that were from apis that normally are typed, "#include <stdlib.h>" but are typed "#include "stdlib.h"" or something like that, and im assuming that certain source codes are copied into the project for compatibility reasons, so i want to be safe here and use the right include)